client/src/api/client.ts

```diff
@@ -0,0 +1,540 @@
+import { errors } from '../util/errors';
+import type * as Types from './components';
+import { operationsByTag } from './components';
+import type { FetcherExtraProps, FetchImpl } from './fetcher';
+import { ApiProviders, getApiUrl } from './providers';
+import type * as Responses from './responses';
+import type * as Schemas from './schemas';
+
+export interface XataApiClientOptions {
+  fetchImpl: FetchImpl;
```

Comment:
Suggestion. We call this `fetch` in the other client. I think it's a bit nicer than `fetchImpl`.

Reply:
Sure thing